ci-operator/config/openshift/cluster-control-plane-machine-set-operator/openshift-cluster-control-plane-machine-set-operator-main.yaml

```diff
@@ -280,7 +280,7 @@ tests:
     env:
       TEST_FILTERS: NonHyperShiftHOST&;cpms;CPMS
       TEST_SCENARIOS: Cluster_Infrastructure
-      TEST_TIMEOUT: "1200"
+      TEST_TIMEOUT: "90"
```

Comment:
What is the unit here, we are going from 1200 to 90 which seems like a significant reduction?

Reply:
Yeah , Joel , I updated title , it was misleading , initially while reviewing we thought it is `seconds` and is less , hence causing failures , realized it wasn't that only , but `PlacementGroups` tests added which were causing it and the time is in `mins` which was wrongly used as `1200` instead should be `90`
So only change in this repo was time, the tests was changed on another repo fixing the failures here.